src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx

```diff
@@ -1,13 +1,244 @@
+import ReassignmentTabPanelContent from './WithdrawalDetails/ReassignmentTabPanelContent';
+import NurseryTransferContent from './WithdrawalDetails/NurseryTransferContent';
+import DeadContent from './WithdrawalDetails/DeadContent';
+import OtherContent from './WithdrawalDetails/OtherContent';
+import { Species } from 'src/types/Species';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  backIcon: {
+    fill: theme.palette.TwClrIcnBrand,
+    marginRight: '4px',
+  },
+  backToWithdrawals: {
+    fontSize: '14px',
+    fontWeight: 500,
+    display: 'flex',
+    textDecoration: 'none',
+    color: theme.palette.TwClrTxtBrand,
+    alignItems: 'center',
+    marginLeft: 0,
+    marginTop: theme.spacing(2),
+  },
+}));
+
+const TABS = ['withdrawal', 'reassignment'];
+
+export interface WithdrawalSummary {
+  id: string;
+  delivery_id: string;
+  withdrawnDate: string;
+  purpose: string;
+  facilityName: string;
+  destinationName: string;
+  plotNames: string;
+  scientificNames: string[];
+  totalWithdrawn: number;
+  hasReassignments: boolean;
+}
 
 type NurseryWithdrawalsDetailsProps = {
   organization: ServerOrganization;
+  species: Species[];
 };
 
-export default function NurseryWithdrawalsDetails(props: NurseryWithdrawalsDetailsProps): JSX.Element {
+export default function NurseryWithdrawalsDetails({
+  organization,
+  species,
+}: NurseryWithdrawalsDetailsProps): JSX.Element {
+  const classes = useStyles();
+  const theme = useTheme();
   const { withdrawalId } = useParams<{ withdrawalId: string }>();
+  const { isMobile } = useDeviceInfo();
+  const contentRef = useRef(null);
+  const [snackbar] = useState(useSnackbar());
+
+  const query = useQuery();
+  const history = useHistory();
+  const location = useStateLocation();
+  const tab = (query.get('tab') || '').toLowerCase();
+  const preselectedTab = TABS.indexOf(tab) === -1 ? 'withdrawal' : tab;
+  const [selectedTab, setSelectedTab] = useState(preselectedTab);
+
+  const [withdrawal, setWithdrawal] = useState<NurseryWithdrawal | undefined>(undefined);
+  const [withdrawalSummary, setWithdrawalSummary] = useState<WithdrawalSummary | undefined>(undefined);
+  const [delivery, setDelivery] = useState<Delivery | undefined>(undefined);
+  const [batches, setBatches] = useState<Batch[] | undefined>(undefined);
+  useEffect(() => {
+    const updateWithdrawal = async () => {
+      const withdrawalResponse = await getNurseryWithdrawal(Number(withdrawalId));
+      if (!withdrawalResponse.requestSucceeded || withdrawalResponse.error) {
+        setWithdrawal(undefined);
+        setDelivery(undefined);
+        setBatches(undefined);
+        snackbar.toastError(withdrawalResponse.error);
+      } else {
+        setWithdrawal(withdrawalResponse.withdrawal);
+        setDelivery(withdrawalResponse.delivery);
+        setBatches(withdrawalResponse.batches);
+      }
+      // get summary information
+      const apiSearchResults = await listNurseryWithdrawals(organization.id, [
```

Comment:
Added this call to get summary information about the withdrawal, which includes `hasReassignments`